### PR TITLE
Create Tooltips for Literature Notes with Supercharged Links.md

### PR DESCRIPTION
**Pull request by chrisgrieser**
Create Tooltips for Literature Notes with Supercharged Links.md

## Checklist

- [x] I have renamed all attached images with descriptive file names (or I did not include any images)
- [x] Before creating a new note, I searched the vault (or I only edited existing notes)
- [x] (Optional) In case I created a new note in the folder `04 - Guides, Workflows, & Courses`, I added a link to the new note(s) in one of the `for {group X}` overviews ([listed here](https://publish.obsidian.md/hub/04+-+Guides%2C+Workflows%2C+%26+Courses/%F0%9F%97%82%EF%B8%8F+04+-+Guides%2C+Workflows%2C+%26+Courses)).

### CODE REVIEW

**Review by argenos — COMMENTED** [review]
Maybe add a link to the plugin so the note shows up in the backlinks?

**Comment by chrisgrieser** [comment]
oh right. thanks, I added a link.

File: 03 - Showcases & Templates/Plugin Showcases/Tooltips for Literature Notes with Supercharged Links.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+# Tooltips for Literature Notes with Supercharged Links
+
+![[Tooltips-for-Literature-Notes-with-Supercharged-Links.gif]]
+
+Don't forget to add "title" (or whichever yaml key you use) to the "tag attributes for styling" in the [[supercharged-links-obsidian|Supercharged Links]] settings.
+
+```css
+/* change "#pdf-annotations" to the tag of your literatur notes */
```
Comment (kometenstaub) [inline 03 - Showcases & Templates/Plugin Showcases/Tooltips for Literature Notes with Supercharged Links.md:15]:
```suggestion
/* change "#pdf-annotations" to the tag of your literature notes */
```

**Review by kometenstaub — CHANGES_REQUESTED** [review]

File: 03 - Showcases & Templates/Plugin Showcases/Tooltips for Literature Notes with Supercharged Links.md (hunk 1 above)
Comment (chrisgrieser) [reply 03 - Showcases & Templates/Plugin Showcases/Tooltips for Literature Notes with Supercharged Links.md]: thanks

**Review by kometenstaub — APPROVED** [review]

**Pull request merged**